onnxruntime/core/providers/cuda/cu_inc/common.cuh

```diff
@@ -230,7 +230,7 @@ __device__ __inline__ T _Gelu(T a) {
 #endif
 
 template <class INT, class INT2>
-static INT CeilDiv(INT a, INT2 b)  // ceil(a/b)
+inline __host__ __device__ INT CeilDiv(INT a, INT2 b)  // ceil(a/b)
```

Comment:
>__device__ [](start = 16, length = 10)

if you use this function in the device, I am wondering if it should be replaced with fast_divmod